crates/turborepo-lib/src/run/summary/execution.rs

```diff
@@ -1,28 +1,168 @@
 use std::fmt;
 
-use chrono::{DateTime, Duration, Local};
-use serde::{ser::SerializeStruct, Serialize};
+use chrono::{DateTime, Duration, Local, SubsecRound};
+use serde::{ser::SerializeStruct, Serialize, Serializer};
 use tokio::sync::mpsc;
-use turbopath::AnchoredSystemPath;
+use tracing::log::warn;
+use turbopath::{AbsoluteSystemPathBuf, AnchoredSystemPath};
+use turborepo_ui::{color, BOLD, BOLD_GREEN, BOLD_RED, MAGENTA, UI};
 
-use crate::run::task_id::TaskId;
+use crate::run::{summary::task::TaskSummary, task_id::TaskId};
 
 // Just used to make changing the type that gets passed to the state management
 // thread easy
 type Message = Event;
 
+fn serialize_datetime<S: Serializer>(
+    date_time: &DateTime<Local>,
+    serializer: S,
+) -> Result<S::Ok, S::Error> {
+    serializer.serialize_i64(date_time.timestamp_millis())
+}
+
+fn serialize_optional_datetime<S: Serializer>(
+    date_time: &Option<DateTime<Local>>,
+    serializer: S,
+) -> Result<S::Ok, S::Error> {
+    let millis = date_time
+        .map(|dt| dt.timestamp_millis())
+        .unwrap_or_default();
+    serializer.serialize_i64(millis)
+}
+
 // Should *not* be exposed outside of run summary module
 /// The execution summary
 #[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
 pub struct ExecutionSummary<'a> {
     // this thread handles the state management
     #[serde(skip)]
     state_thread: tokio::task::JoinHandle<SummaryState>,
     #[serde(skip)]
     sender: mpsc::Sender<Message>,
     command: String,
-    package_inference_path: Option<&'a AnchoredSystemPath>,
-    started_at: DateTime<Local>,
+    success: usize,
+    failed: usize,
+    cached: usize,
+    attempted: usize,
```

Comment:
All of these are tracked already in the `SummaryState` method that gets returned by calling `finish`.
```suggestion
```